packages/ui/src/component/UICanvas.ts

```
@@ -297,7 +305,7 @@
   /**
    * @internal
    */
-  _updateSortDistance(cameraPosition: Vector3): void {
+  _updateSortDistance(isOrthographic: Boolean, cameraPosition: Vector3, cameraForward: Vector3): void {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)